common/src/sql/dbinit.sql

```diff
@@ -65,7 +65,8 @@ CREATE TABLE omicron.public.sled (
     rcgen INT NOT NULL,
 
     ip INET NOT NULL,
-    port INT4 NOT NULL
+    port INT4 NOT NULL,
```

Comment:
Might be worth a comment indicating that "IP + port" are the address of the sled agent, and "last_used_address" describes the portion of the `/64` the sled is responsible.

(Seems plausible to think "port" is associated with "last_used_address", which it is not)

Reply:
Good call, will do.